lib/views/pages/account_settings/account_settings.dart

```diff
@@ -16,7 +16,6 @@ class AccountSettings extends HookConsumerWidget {
 
```

Comment:
@rdelrosariojr cc: @inahsanjuan 
curios lang ko why its under a folder? 🙇

Reply:
@AubreyV I was thinking ma'am since there are different pages for account settings like update account info. update, currency, etc. So Decided to group them nalang

Reply:
@rdelrosariojr Ahhh yes! OK gets ko na! Thanks Rob